### PR TITLE
New modify

### PR DESCRIPTION
**Pull request by fosabdul**
New modify branch made,

### CODE REVIEW

**Comment by MinaYimam** [comment]
This worked for me, I am going to merge it!

**Review by MinaYimam — COMMENTED** [review]
This looks good. It works!

File: README.md [hunk 1]
```diff
@@ -3,8 +3,12 @@
 Basic python book reading list application. 
 For practicing teamwork and GitHub collaboration. 
 This is a program that lets user add, delete, search, 
+<<<<<<< HEAD
```
Comment (claraj) [inline README.md:6]: This is a merge conflict marker, it should be removed

File: README.md [hunk 2]
```diff
@@ -3,8 +3,12 @@
...
+show and, read books. Also this program adds all the books in the 
+=======
```
Comment (claraj) [inline README.md:8]: and this

File: README.md [hunk 3]
```diff
@@ -3,8 +3,12 @@
...
 show and, read books.
 Also this program adds all the books in the 
+>>>>>>> 12532dcda82be6657cb2c412ad72696cbbbea28b
```
Comment (claraj) [inline README.md:11]: and this.

**Pull request merged**